src/engine/controls/cuecontrol.cpp

```diff
@@ -2430,6 +2526,11 @@ void CueControl::slotLoopEnabledChanged(bool enabled) {
         return;
     }
 
+    if (pSavedLoopControl && pSavedLoopControl->getType() != mixxx::CueType::Loop) {
+        slotLoopReset();
+        return;
+    }
```

Comment:
```suggestion
    if (pSavedLoopControl->getType() != mixxx::CueType::Loop) {
        slotLoopReset();
        return;
    }
```
This is not needed, above we already return if there's no `pSavedLoopControl`